### ISSUE TITLE
Problem with sequential model training

### ISSUE THREAD

**Issue by hogeman2** [keras-team/keras#10825]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [v ] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [ v] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

I'm trying to train sequential model. 
I defined two model like this
def model1 :
  inputs = Input(input_shape=(256,256,3))
  output = Conv2d(channels = ch, output=(w,h) , name='model1_conv')(inputs)
  model1 = Model(inputs = [inputs], outputs= output)

def real_model:
  input_tensor = Input(input_shape=(256,256,3))
  F1 = model1(inputs=(256,256,3))
  fm = Model(inputs=F1.input, outputs=F1.output)
  
  fm_out1 = fm(data1)
  fm_out2 = fm(data2)
  
   conc_out = concate(fm_out1,fm_out2) 
  conc_out = conv2d(name='real_conv')(conc_out)  
  conc_out = sigmoid()(conc_out) 
  model = Model(inputs=input_tensor, outputs= conc_out)
  
and the results seems that only the conv2d layer in real_model ('real_conv') is trainied and model1 weights  is not changing at all.

I don't know how to solve this. Is there any one who can solve this ? 

Thank for reading.

**Comment by greed2411** [comment]
Just a hunch but 
i found these two things to be bugging:

1. `fm = Model(inputs=F1.input, outputs=F1.output)` you have given `F1.input` and `F1.output` instead of `F1.inputs` and `F1.outputs`, maybe i'm wrong with this.
2.  `concate` instead of [`concatenate`](https://keras.io/layers/merge/#concatenate_1) in the line `conc_out = concate(fm_out1,fm_out2)`

**Comment by hogeman2** [comment]
@greed2411 Thanks for reply.  above one is pseudo. so real implementation I'm using  written those functions you wrote. I should have mentioned it , sorry.
